.github/workflows/cron.create_snapshot.yml

```
@@ -0,0 +1,174 @@
+name: '[CRON] Create Snapshot'
+on:
+  workflow_dispatch: # run from the UI
+  schedule:
+    - cron: '0 4 * * *'
+
+jobs:
+  init-and-validate:
+    name: Init and validate
+    runs-on: ubuntu-latest
+    if: vars.ENABLE_SNAPSHOT_GENERATION == 'true'
```

```suggestion
    if: vars.SF_ENABLE_SNAPSHOT_GENERATION == 'true'
```
 Lik naming convention? 😊